src/browser/bounds_changed_state_tracker.js

```diff
@@ -130,7 +130,7 @@ function BoundsChangedStateTracker(uuid, name, browserWindow) {
         var currentBounds = getCurrentBounds();
         var cachedBounds = getCachedBounds();
         var boundsCompare = compareBoundsResult(currentBounds, cachedBounds);
-        var stateMinMax = boundsCompare.state && currentBounds.state !== 'normal'; // maximized or minimized
+        var stateMin = boundsCompare.state && currentBounds.state === 'minimized';
```

Comment:
Relaxing this condition so it behaves more like v5. We only want to restrict deferred events to minimized windows.